build.gradle.kts

```diff
@@ -83,10 +83,10 @@ extra.apply {
    val projectVersion = project.version as String
    val uppercaseProjectName = projectName.substring(0,1).toUpperCase() + projectName.substring(1)
    set("uppercaseProjectName", uppercaseProjectName)
-   set("appDirname", "$buildDir/dist/" + projectName + ".app")
+   set("appDirname", "$buildDir/dist/" + uppercaseProjectName + ".app")
```

Comment:
Not that it's crucial or related but I think there's variable naming inconsistency. You have  `uppercaseProjectName` but `appDirname` (occurs multiple times with various vars). Is it intentional? If not, what about sticking to one form across the whole file?

Reply:
It is the inconsistency of consistency! I was trying to follow google java code standards. I have variables such as dmgFilename, which is correct because filename is an English word. appDirname was written to match that, but dirname is not an English word; so that does not actually conform to the standard. You are correct that it should be appDirName.

While on the naming topic, there is also jpackagecmd that existed before I named other things. Should I also change that to jPackageCmd?

Reply:
Done.